shared/src/commonMain/kotlin/App.kt

```diff
@@ -6,18 +9,39 @@ import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
 
 @Composable
 internal fun App() {
+
     MaterialTheme {
         var text by remember { mutableStateOf("Hello, World!") }
 
-        Button(onClick = {
-            text = "Hello, ${getPlatformName()}"
-        }) {
-            Text(text)
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(
+                text = text,
+                style = TextStyle(
+                    textAlign = TextAlign.Center,
+                    fontSize = 30.sp
+                ),
+            )
+            Button(
+                onClick = {
+                    text = getMokoResourcesText()
+                }
+            ) {
+                Text("Get moko text")
+            }
         }
     }
 }
 
-expect fun getPlatformName(): String
\ No newline at end of file
+expect fun getMokoResourcesText(): String
```

Comment:
we should not use expect, all expects should be in moko-resources lib

Reply:
deleted, return previous name